parcel/parcel.jl

```diff
@@ -0,0 +1,216 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+
+const CMT = CM.CommonTypes
+const CMO = CM.Common
+const CMI = CM.HetIceNucleation
+const CMP = CM.Parameters
+
+FT = Float64
```

Comment:
I would try avoiding this. I'll sprinkle some notes below on things you may need to change

Reply:
You mean avoiding adding this example to the documentation? Or avoiding defining FT here?

Reply:
Avoid defining FT like this

Reply:
Ah, maybe just delete this definition and use `toml_dict = CP.create_toml_dict(Float64; dict_type = "alias")` below?